magento/pwa-studio
# Scroll completed section to the top after every checkout step.

## Description

Scroll the view to each completed section after every checkout step to allow the user to check the progress and the data they have entered.

After the Shipping Information step is done

![image](https://user-images.githubusercontent.com/35203638/111252008-3e562900-85de-11eb-8f2b-681c02dbcf61.png)

![image](https://user-images.githubusercontent.com/35203638/111252038-4d3cdb80-85de-11eb-93db-20596e1da4e5.png)

After the Shipping Method step is done

![image](https://user-images.githubusercontent.com/35203638/111252141-7c534d00-85de-11eb-82b6-38c078544514.png)

![image](https://user-images.githubusercontent.com/35203638/111252068-5cbc2480-85de-11eb-9cef-be53cdb2d7f7.png)

## Related Issue
Closes PWA-723

### Verification Stakeholders
@schensley 
@dpatil-magento 

### Verification Steps
1. Go to the deployed app's checkout page.
2. After each step is completed, you should notice that the window should scroll to the top of the view.

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
